cypress/e2e/tests/pages/global-settings/banners.spec.ts

```diff
@@ -111,12 +120,23 @@ describe('Banners', { testIsolation: 'off' }, () => {
 
       // Check in session
       bannersPage.banner().should('be.visible').then((el) => {
-        expect(el).to.have.attr('style').contains(`color: ${ settings.bannerTextColor.newRGB }`);
         expect(el).to.have.attr('style').contains(`background-color: ${ settings.bannerBackgroundColor.newRGB }`);
         expect(el).to.have.attr('style').contains(`text-align: ${ settings.textAlignment.new.toLowerCase() }`);
-        expect(el).to.have.attr('style').contains(`ext-decoration: ${ settings.textDecoration.toLowerCase() }`);
-        expect(el).to.have.attr('style').contains(`font-size: ${ settings.fontSize.new }`);
+        expect(el).to.have.attr('style').contains(`text-decoration: ${ settings.textDecoration.toLowerCase() }`);
       });
+      bannersPage.banner().find('div').then((el) => {
+        const win = cy.state('window');
+        const styles = win.getComputedStyle(el[0]);
+
+        expect(styles.getPropertyValue('color')).to.equal(settings.bannerTextColor.newRGB);
+        expect(styles.getPropertyValue('font-weight')).to.equal(settings.fontWeight);
+        expect(styles.getPropertyValue('font-style')).to.equal(settings.fontStyle);
+        expect(styles.getPropertyValue('font-size')).to.equal(settings.fontSize.new);
```

Comment:
I wanted to move these checks deeper. The original tests didn't catch that we weren't styling the child element correctly.